prac_05/word_occurrences.py

```diff
@@ -0,0 +1,23 @@
+"""
+Word Occurrences
+Estimate: 50 minutes
+Actual:   76 minutes
+"""
+
+string_to_count = {}
+string = input("Text: ")
+words = string.split()
+print(words)
+for word in words:
+    try:
+        string_to_count[word] += 1
+    except KeyError:
+        string_to_count[word] = 1
+
+words = list(sorted(string_to_count.keys()))
```

Comment:
What does this line do? If it is meant to sort the words in the dictionary, we can do:
>> string_to_count = dict(sorted(string_to_count))